e2e/test/scenarios/models/model-actions.cy.spec.js

```diff
@@ -14,7 +14,11 @@ import {
   queryWritableDB,
 } from "e2e/support/helpers";
 
-import { SAMPLE_DB_ID, USER_GROUPS, WRITABLE_DB_ID } from "e2e/support/cypress_data";
+import {
```

Comment:
This is the result from running `prettier`, we missed running it on files under `e2e` since , I believed, https://github.com/metabase/metabase/pull/28749